package.json

```
@@ -82,58 +82,58 @@
 		"@wordpress/api-fetch": "6.23.1",
 		"@wordpress/autop": "3.26.1",
 		"@wordpress/blob": "3.26.1",
-		"@wordpress/block-directory": "4.3.3",
-		"@wordpress/block-editor": "11.3.3",
-		"@wordpress/block-library": "8.3.3",
+		"@wordpress/block-directory": "4.3.4",
+		"@wordpress/block-editor": "11.3.4",
+		"@wordpress/block-library": "8.3.4",
 		"@wordpress/block-serialization-default-parser": "4.26.1",
 		"@wordpress/blocks": "12.3.2",
-		"@wordpress/components": "23.3.2",
+		"@wordpress/components": "23.3.3",
 		"@wordpress/compose": "6.3.2",
 		"@wordpress/core-data": "6.3.2",
-		"@wordpress/customize-widgets": "4.3.3",
+		"@wordpress/customize-widgets": "4.3.4",
 		"@wordpress/data": "8.3.2",
 		"@wordpress/data-controls": "2.26.2",
 		"@wordpress/date": "4.26.2",
 		"@wordpress/deprecated": "3.26.1",
 		"@wordpress/dom": "3.26.1",
 		"@wordpress/dom-ready": "3.26.1",
-		"@wordpress/edit-post": "7.3.3",
-		"@wordpress/edit-site": "5.3.3",
-		"@wordpress/edit-widgets": "5.3.3",
-		"@wordpress/editor": "13.3.3",
+		"@wordpress/edit-post": "7.3.4",
+		"@wordpress/edit-site": "5.3.4",
+		"@wordpress/edit-widgets": "5.3.4",
+		"@wordpress/editor": "13.3.4",
 		"@wordpress/element": "5.3.1",
 		"@wordpress/escape-html": "2.26.1",
 		"@wordpress/experiments": "0.8.1",
```

This package is no longer in use after it got renamed to `@wordpress/private-apis`:
```suggestion
-		"@wordpress/experiments": "0.8.1",
```

This will require changes in the lock file, too.

---

Good catch! I'll update now!